packages/playwright-core/src/server/browserType.ts

```diff
@@ -294,10 +295,16 @@ export abstract class BrowserType extends SdkObject {
     }
   }
 
+  _rewriteStartupError(error: Error): Error {
+    if (!isProtocolError(error))
+      return error;
+    return this._doRewriteStartupError(error);
```

Comment:
Are you sure it is always a protocol error? Could it be something else, when process suddenly exits for example?